.env.test

```diff
@@ -1,8 +1,8 @@
 DATABASE_URL=postgresql://postgres:postgres@localhost:5432/hollo_test
 SECRET_KEY="test_determinist_key_DO_NOT_USE_IN_PRODUCTION"
 
-LOG_LEVEL=debug
-LOG_QUERY=true
+# LOG_LEVEL=debug
+# LOG_QUERY=true
```

Comment:
Whilst these can be useful in debugging tests, they also make the output harder to read (and it's already pretty hard)

Reply:
Do we have any flag to determine if it's testing? We should adjust *src/logging.ts* for the testing environment.